src/components/Form/FieldPassword/index.tsx

```diff
@@ -38,6 +39,7 @@ export const FieldPassword = <
 >(
   props: FieldPasswordProps<TFieldValues, TName>
 ) => {
+  const { t } = useTranslation(['common', 'components']);
```

Comment:
```suggestion
  const { t } = useTranslation(['components']);
```
il est pas utile le common ?